### ISSUE TITLE
At 200% zoom buttons present under 'Remaining_Controls_Testing' are not visible: A11y_.NET CoreWinforms_MainForm_Zoom

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11457]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev1; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG1.4.4; #DesktopApp; #Win11; #FTP; #Zoom;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Prerequisites:

1. Display resolution: 1280*768
2. Scale-> Custom scaling 200%

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. Do the resize resolution and observe that At 200% zoom buttons present under 'Remaining_Controls_Testing' are not visible.

### Actual Result:
At 200% zoom buttons present under 'Remaining_Controls_Testing' are not visible.

**Similar issue is also observed in below scenario:
Issue 1: ToolStripContainer_Bottomtool bar**

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "MenuToolbars_Controls:Testing the controls under Menu Toolbars Tab" and hit ENTER key.
4. TAB to ToolStripContainer button and hit ENTER key.
5. TAB to "Button1" present at the botoom
6. Observe that at 200% zoom "Button1" present at the bottom is not visible.

**Issue2: Print_ Preference**

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Printing_Controls:Testing the controls under Printing Tab" and hit ENTER key.
4. TAB to Print button and hit ENTER key.
5. TAB to More setting button and hit ENTER key.
6. "Printing Preference" Screen would appear.
7. Observe that at 200% "Advance", "ok" and "Cancel" buttons are not visible.

### Expected Result:
At 200% zoom buttons present under 'Remaining_Controls_Testing' should be clearly visible.

### User Impact:
Low vision users will not be able to see the controls clearly and will not be getting information related to those.

### Attachment
![11457_A11y_ NET CoreWinforms_MainForm_Zoom](https://github.com/dotnet/winforms/assets/90901519/edaa21b2-0caa-40bb-928b-b318682393f6)

Issue1: 
![11457_Issue 1_ Toolstripcontainer](https://github.com/dotnet/winforms/assets/90901519/e63fd28a-658e-4c69-970c-59d1bac30588)

Issue2: 
![11457_Issue2_ Print_ Preference](https://github.com/dotnet/winforms/assets/90901519/a75d2507-ddd0-4ecf-8fa8-09418ccdb1bd)

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by MandiMan** [comment]
This issue does not repro on screens with a display resolution of 3840x2160 and a scale of 200%.
![image](https://github.com/dotnet/winforms/assets/108860782/597fe93d-0280-4715-997c-723bd5e17e96)
![image](https://github.com/dotnet/winforms/assets/108860782/a03aec74-fc3f-45a2-b144-736982176706)
![image](https://github.com/dotnet/winforms/assets/108860782/3f57d6ce-dcee-48a6-b71b-7371805e137d)

**Comment by merriemcgaw** [comment]
Also note that this is a Windows dialog not ours. This should fall under the Printing Dialog exception we already have.

**Comment by PoojaNamde** [comment]
@MandiMan Issue should be fixed for 1280 * 768 recommended display resolution as most of the users will be using this resolution. So could you fix it for and assign back to me.

**Comment by merriemcgaw** [comment]
@PoojaNamde this is a screen that is owned by Windows and we have no control over it. Previously we had been excluding certain printer related screens because we already have an exception. This should fall under the existing exception for Windows Printing APIs that we have.

**Comment by PoojaNamde** [comment]
@merriemcgaw, as per above comment as there is existing windows exception for printing screen, we will exclude that issue from this bug. Could you please fix other two issues mentioned in the bug.

**Comment by merriemcgaw** [comment]
@Olina-Zhang can your team address these issues? Please go through the app and make changes where necessary to support (in System Aware) 200% scaling at the display and scale size mentioned in the prerequisites. You can add new screens and/or tabs to make sure everything fits the way it should. I think there are a few screens where we should do stuff.

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-20-24;

Verified this issue in the latest environment, issue is fixed hence closing the bug.

![Screenshot (1106)](https://github.com/dotnet/winforms/assets/90901519/85c90348-9b8c-453d-821c-a3b62c0f44fd)
![Screenshot (1107)](https://github.com/dotnet/winforms/assets/90901519/50449ab5-4af1-4cfa-a26f-e198a0e11198)
